src/pages/accounts/AccountDetails.tsx

```diff
@@ -1,451 +1,279 @@
-                    contact_name: accountDetails?.contact_name,
-                    teams: accountDetails?.teams || [],
-                    assigned_to: accountDetails?.assigned_to || [],
-                    tags: accountDetails?.tags || [],
-                    account_attachment: accountDetails?.account_attachment || null,
-                    website: accountDetails?.website,
-                    status: accountDetails?.status,
-                    lead: accountDetails?.lead?.title || '',
-                    contacts: accountDetails?.contacts || [],
-        
-                }, id: state?.accountId,
-                contacts: state?.contacts || [], status: state?.status || [], tags: state?.tags || [], users: state?.users || [], countries: state?.countries || [], teams: state?.teams || [], leads: state?.leads || []
-            }
-        }
-        )
-    }
+      const Header = {
+        Accept: 'application/json',
+        'Content-Type': 'application/json',
+        Authorization: localStorage.getItem('Token') ?? '',
+        org: localStorage.getItem('org') ?? '',
+      };
 
-    const backbtnHandle = () => {
-        navigate('/app/accounts')
-    }
+      try {
+        const res = await fetchData(
+          `${AccountsUrl}/${accountId}/`,
+          'PATCH',
+          JSON.stringify(data),
+          Header
+        );
+        if (!res.error) {
+          await fetchAccount(); // Re-fetch account data after updating
+        } else {
+          console.error('Error updating description:', res.error);
+        }
+      } catch (error) {
+        console.error('Request failed:', error);
+      }
+    };
 
-    interface AssignedToItem {
-        id: string;
-    }
-    const isAssignedToAccount = (accountDetails?.assigned_to ?? []).some(item => item.user_details.id === userId);
-    const isSalesManager = userRole === "SALES MANAGER";
-    const isCreatedByUser = accountDetails?.created_by.id === userId;
-    const showEditButton = !isSalesManager || isAssignedToAccount || isCreatedByUser;
+    submitForm();
+  };
+const test = true
+  // Show loading message if account data is not yet available
+  // if (test) {
+  //   return (<Box style={{height:1000, width:1000, backgroundColor: "red"}}>Loading...</Box>)
+  // }
+  console.log("account", account)
+  if (!account) {
+    return (<Box>Loading...</Box>)
+  }
 
-    const module = 'Accounts'
-    const crntPage = 'Account Details'
-    const backBtn = 'Back To Accounts'
+  return (
+    <Box sx={{ mt: '60px'}}>
+      <CustomToolbar
+        sx={{
+          display: 'flex',
+          flexDirection: 'row',
+          alignItems: 'center',
+          justifyContent: 'space-between',
+        }}
+      >
+        {(userRole === 'ADMIN' ||
+          account.created_by.id === userId ||
+          account.account_obj.assigned_to.some(
+            (assignee: any) => assignee.user_details.id === userId
+          )) && (
+          <DynamicModal
+            page="Deals"
```

Comment:
In `DynamicModel` the `page` prop is important because based on that it renders the correct form. 
Now we're in accounts pages, so this one needs to change to `page="Accounts"`